Microsoft/WebTemplateStudio
# App Service - update application settings

This change is required to update the application setting of App Service web app.

When Cosmos DB is selected, we generate a .env file which contains the connection string to the database. If App Service and Cosmos DB are selected, we want the App Service application settings to also contains the connection string.

**To test:**
1. Launch the extension in preview mode
2. Add App Service and Cosmos DB (test for both Mongo and SQL)
3. Create the project
4. Once everything is deployed, click "Yes" in the prompt that updates the connection string in the .env file
5. Go onto Azure portal (https://ms.portal.azure.com/#home)
6. Go into the App Service created -> Configuration -> Application settings
![image](https://user-images.githubusercontent.com/20333350/62829923-40ae0100-bbb9-11e9-8d35-eca0620246c1.png)
7. Make sure it is the same as the content of the .env file
